src/serializers/infer.rs

```diff
@@ -483,6 +485,7 @@ pub(crate) fn infer_serialize_known<S: Serializer>(
             pydantic_serializer.serialize(serializer)
         }
         ObType::Dataclass => serialize_dict!(dataclass_to_dict(value).map_err(py_err_se_err)?),
+        ObType::Uuid => serialize_dict!(uuid_to_dict(value).map_err(py_err_se_err)?),
```

Comment:
Similarly here I think we just want the string serialized?

Reply:
Done